src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,6 +6,7 @@
 public class CliSyntax {
 
     /* Prefix definitions */
+    public static final Prefix PREFIX_UID = new Prefix("id/");
```

Comment:
I think this prefix currently is to be used in mark, delete, edit commands those, but not others, maybe?

Reply:
This is just to implement the uid into the existing codebase, it will change based upon how the individual commands are written